scripts/benchmarks_ci.py

```diff
@@ -243,7 +243,8 @@ def __main(args: list) -> int:
 
         if args.upload_to_perflab_container:
             import upload
-            upload.upload(globpath, upload_container, UPLOAD_QUEUE, UPLOAD_TOKEN_VAR, UPLOAD_STORAGE_URI)
+            upload_code = upload.upload(globpath, upload_container, UPLOAD_QUEUE, UPLOAD_TOKEN_VAR, UPLOAD_STORAGE_URI)
```

Comment:
It may be better to only return on failure incase things get added that run after upload in the future.